### ISSUE TITLE
The Toolbar overflow menu button has no caption and is announced by the screen readers as "blank"

### ISSUE THREAD

**Issue by veselints** [telerik/kendo-ui-core#5392]
### Bug report
The Toolbar overflow menu button has no caption and is announced by the screen readers as "blank"

### Reproduction of the problem
* Run the following demo: https://demos.telerik.com/kendo-ui/toolbar/keyboard-navigation
* Navigate to the Overflow button - it will be announced as "blank".

### Expected/desired behavior
The Overflow button should be announced properly

### Environment
* **Kendo UI version:** 2019.3.1023
* **Browser:** [all]

**Comment by mparvanov** [comment]
(2020.2.617) The Overflow button is not announced by the screen readers since there is no relevant aria attribute: https://dojo.telerik.com/emoteBan
A possible resolution might be adding aria-label attribute to the button:

 <div tabindex="0" class="k-overflow-anchor k-button" style="visibility: visible;" **aria-label="Show options"**><span class="k-icon k-i-more-vertical"></span></div>

https://dojo.telerik.com/emoteBan/2

**Comment by GaloisGirl** [comment]
@mparvanov yes, that is what I had to do. 

However, since the toolbar is listed as WCAG compliant on https://docs.telerik.com/kendo-ui/accessibility/section-508-wcag , this should be built-in.
